lms/static/js/verify_student/views/make_payment_step_view.js

```diff
@@ -105,10 +105,20 @@ var edx = edx || {};
                 self._getProductText( templateContext.courseModeSlug, templateContext.upgrade )
             );
 
-            // create a button for each payment processor
-            _.each(processors.reverse(), function(processorName) {
-                $( 'div.payment-buttons' ).append( self._getPaymentButtonHtml(processorName) );
-            });
+            if (processors.length === 0) {
+                // No payment processors are enabled at the moment, so show an error message
+                this.errorModel.set({
+                    errorTitle: gettext('No available payment processors'),
```

Comment:
@srpearce yes, that text is part of a test and will need to be updated as well. To give you some context, this is an error message displayed when all payment options on the pre-payment page are disabled (because the processors themselves are having problems). This message is meant to be an acknowledgement that the user is unable to select a payment option, and that the problem should be resolved soon.